### ISSUE TITLE
link rhdf5 to Rhdf5lib

### ISSUE THREAD

**Issue by mikejiang** [Huber-group-EMBL/Rhdf5lib#2]
There is no `rhdf5` github repo, so I just post it here. Maybe it make more sense now to remove the hdf5 c source from `rhdf5` package to reduce the redundancy?

**Comment by grimbough** [comment]
The **rhdf5** repo is at [https://github.com/grimbough/rhdf5](https://github.com/grimbough/rhdf5)

I totally agree, and it doesn't make sense to have two versions of _libhdf5_ floating around in Bioconductor. 
 This is something I started looking into a while back, (there's a branch in my old **rhdf5** repo [here](https://github.com/grimbough/archive-rhdf5/tree/Rhdf5lib))

It didn't go as smoothly as I'd hoped (I thought it would be easy), and there is clearly at least one major change in behaviour between the two versions of _libhdf5_.  This meant the every time `H5close()` was called no more commands would work and you got a wall of error messages until the package was reloaded - that's pretty a pretty dramatic change and Bernd used `H5close()` liberally in the vignette.

That's fairly easy to work around, but I was worried there might be some more subtle changes that I wouldn't notice so easily, so I've spent a while implementing lots of unit tests in  **rhdf5** to try and catch anything else.  That's nearly done now, so and my intention is to remove the source and link against **Rhdf5lib** in January.
